recipes-debian/apt/apt_debian.bbappend

```diff
@@ -0,0 +1 @@
+DEPENDS += "gnutls"
```

Comment:
meta-debian-extended doesn't provide bbappend basically. This should go to apt_debian.bb.

Reply:
I moved this fix to apt_debian.bb.